test/client/index.test.js

```diff
@@ -83,18 +83,6 @@ describe("index", () => {
     expect(socket.mock.calls[0]).toMatchSnapshot();
   });
 
-  test("should run onSocketMessage.hot", () => {
-    onSocketMessage.hot();
-
-    expect(log.log.info.mock.calls[0][0]).toMatchSnapshot();
```

Comment:
we removed these logs from `onSocketMessage.hot()` because otherwise, we would have duplicate logs spamming the console.